### ISSUE TITLE
English-german Test and Dev Blue metrics show 0.0 

### ISSUE THREAD

**Issue by mohammedayub44** [tensorflow/nmt#292]
My test and dev blue metric shows as 0.0. I replicated the whole setup on windows machine. Using the pre-trained 8 layer model. Not sure if there is something missing in the setup doc or I'm passing the wrong files to the training argument. I followed this link. https://www.tensorflow.org/tutorials/seq2seq  

This is the command I'm running:
`python -m nmt.nmt --src=de --tgt=en --hparams_path=standard_hparams\wmt16_gnmt_8_layer.json --out_dir=D:\Projects\nmt_model\German_English_test --vocab_prefix=D:\Projects\nmt_data\german_english\vocab.bpe.32000 --train_prefix=D:\Projects\nmt_data\german_english\train.tok.clean.bpe.32000 --dev_prefix=D:\Projects\nmt_data\german_english\newstest2014.tok.bpe.32000 --test_prefix=D:\Projects\nmt_data\german_english\newstest2016.tok.bpe.32000`

Attached are my dev and test files (All of them show as <UNK> for some wierdreason) 

[output_dev.txt](https://github.com/tensorflow/nmt/files/1874154/output_dev.txt)
[output_test.txt](https://github.com/tensorflow/nmt/files/1874155/output_test.txt)

Here is are my vocab file: (both are basically the same file duplicated as suggested in the setup link. I had to remove duplicates as I was getting hash table error and open and re-save them as UTF-8 because of decode errors from python)
[vocab.bpe.32000.de.txt](https://github.com/tensorflow/nmt/files/1874161/vocab.bpe.32000.de.txt)
[vocab.bpe.32000.en.txt](https://github.com/tensorflow/nmt/files/1874165/vocab.bpe.32000.en.txt)

Here is the log file:
[log_1522787979.txt](https://github.com/tensorflow/nmt/files/1874171/log_1522787979.txt)

Any help appreciated. Thanks !
